lua/neogit/lib/git/status.lua

```diff
@@ -3,24 +3,10 @@ local git = {
   stash = require("neogit.lib.git.stash")
 }
 local a = require 'plenary.async_lib'
-local async, await, await_all, future = a.async, a.await, a.await_all, a.future
+local async, await, await_all = a.async, a.await, a.await_all
 local util = require("neogit.lib.util")
 local Collection = require('neogit.lib.collection')
 
-local function marker_to_type(m)
```

Comment:
Why is this getting removed? Can we configure luacheck to tell us about unused functions, but not remove it?

Reply:
You can ignore the warning. But better would be to just remove everything unused.

Reply:
👍 to removing the function, I was just a bit concerned that some code might get removed that is accidently unused.

Reply:
-- luacheck: ignore marker_to_type
would silence the warning